test/built-ins/Temporal/PlainDateTime/from/month-data-needed.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.from
+description: A plain object argument needs to specify a month
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+TemporalHelpers.assertPlainDateTime(
```

Comment:
I think you have a few similar tests in `argument-object.js` - please take a look

Reply:
you're right -- I removed a couple of redundant tests.